src/Microsoft.ServiceFabric.Actors/Runtime/KvsActorStateProvider.cs

```diff
@@ -48,8 +50,12 @@ public KvsActorStateProvider()
         /// <param name="localStoreSettings">
         /// A <see cref="LocalStoreSettings"/> that describes local key value store settings.
         /// </param>
-        public KvsActorStateProvider(ReplicatorSettings replicatorSettings = null, LocalStoreSettings localStoreSettings = null)
-            : this(replicatorSettings, localStoreSettings, false, null)
+        /// <param name="keyValueStoreReplicaSettings"> Key value store settings.</param>
+        public KvsActorStateProvider(

```

Comment:
I believe you rebased with develop?

Reply:
No. Just cherry picked your commit for this change